exercises/perfect-numbers/src/example.c

```diff
@@ -0,0 +1,28 @@
+#include "perfect_numbers.h"
+
+static int aliquot_sum(int n)
+{
+   int result = 0;
+   for (int i = 1; i < n; ++i) {
+      if ((n % i) == 0) {
+         result = result + i;
+      }
+   }
+   return result;
+}
+
+kind classify_number(int n)
+{
+   if (n <= 0) {
+      return error;
+   }
+   int buf = aliquot_sum(n);
+   if (buf == n) {
```

Comment:
Again, only a very small detail; I would use the rule that: an `if` should end with an `else` if practical. So this would become: 
```C
if (buf > n) { 
    return abundant_number; 
} else if (buf < n) {
    return deficient_number;
} else {
    return perfect_number;
}
```
Though I guess this would be somewhat pedantic coding! 😆  (It comes from the MISRA rule regarding `if ... else if` usage IIRC)